sunpy/cm/color_tables.py

```diff
@@ -17,7 +18,7 @@ def _mkx(i, steps, n):
     """ Generate list according to pattern of g0 and b0. """
     x = []
     for step in steps:
-        x.extend(range(i, step + n, n))
+        x.extend(list(range(i, step + n, n)))
```

Comment:
Why is the `list` needed here? I've tried it on py2 and 3 and works fine in both without it...